cellpack/autopack/writers/__init__.py

```diff
@@ -175,7 +175,8 @@ def save_as_simularium(self, env, all_ingr_as_array, compartments):
         file_name = env.helper.writeToFile(
             env.result_file, env.boundingBox, env.name, env.version
         )
-        autopack.helper.post_and_open_file(file_name)
+        if env.config_data.get("number_of_packings", 1) <= 1:
```

Comment:
Thanks for incorporating this! However, I think keeping the `upload_results` option is still a good idea and we should only fall back on `number_of_packings` check if `upload_results` is missing from the config. 

My suggestion was:
`if env.config_data is None or env.config_data.get("upload_results", env.config_data.get("number_of_packings", 1) <= 1):`
which only uploads results and opens a tab if following conditions are met:
1) Config data is missing
Or
2) If `upload_results` is present in config data, it uses whatever value is specified: this allows the user to disable the opening of tabs automatically if they wish. Having this option would be useful in my opinion.
Or
3) If `upload_results` is missing in config data, we check `number_of_packings` and only upload/post if it is less than 1.
Let me know if this makes sense.